src/Text/Pandoc/Readers/DocBook.hs

```diff
@@ -1076,7 +1076,10 @@ strContentRecursive = strContent .
   (\e' -> e'{ elContent = map elementToStr $ elContent e' })
 
 elementToStr :: Content -> Content
-elementToStr (Elem e') = Text $ CData CDataText (strContentRecursive e') Nothing
+elementToStr (Elem e') =
+  case qName (elName e) of
+        n | n == "xref" || n == "co" -> Text $ CData CDataText (show e) Nothing
```

Comment:
The show is just a placeholder until I find out a function to stringify the [`Element`](https://hackage.haskell.org/package/xml-conduit-1.9.1.1/docs/Text-XML.html#t:Element).